prisma/schema.prisma

```diff
@@ -238,3 +241,34 @@ model SourceApi {
 
   @@map(name: "sources_api")
 }
+
+model DiscussionThread {
+  id           String     @id @default(uuid())
+  title        String?
+  number       Int
+  user         User       @relation(fields: [userId], references: [id])
+  userId       String     @map(name: "user_id")
```

Comment:
Is this redundant since the user should be the `user` in the first `DiscussionItem`?